src/components/Dashboard/FilingHistoryList/CompletedIa.vue

```diff
@@ -23,10 +23,9 @@ import { navigate } from '@/utils'
 @Component({})
 export default class CompletedIa extends Vue {
   @Getter readonly getEntityName!: string
-
+  @Getter readonly getMyBusinessRegistryUrl!: string
   protected returnToMyBusinessRegistry (): void {
```

Comment:
Whitespace please.

Reply:
Good call, I'll fix it

Reply:
I meant whitespace between getters and local methods.

Reply:
Ah, got it, thanks for the clarification.  I'll fix it.